spec/lib/tasks/decidim/db/surveys/clear_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "rake decidim:db:surveys:clean", type: :task do
```

Comment:
```suggestion
  let(:stub) { Decidim::SurveysService.new }

  before do
    allow(Decidim::SurveysService).to receive(:new).and_return stub
    allow(stub).to receive(:clear).and_return(true)
  end

  it "preloads the Rails environment" do
    expect(task.prerequisites).to include "environment"
  end

  it "invokes the 'clear' method" do
    task.execute
    expect(stub).to have_received(:clear)
  end
```

I think it is easier to read when stubbing is in a `before` block, what do you think ?